csharp/FigiApiCsharpExample/BloombergArrayResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FigiApiCsharpExample
+{
+    public class BloombergArrayResponse
```

Comment:
Can we replace `Bloomberg` -> `OpenFIGI`

Reply:
Of course!